### PR TITLE
Fix project hydration page

### PR DESCRIPTION
**Pull request by MohammadPCh**
- #585

### CODE REVIEW

File: pages/project/[projectIdSlug]/index.tsx [hunk 1]
```diff
@@ -1,11 +1,16 @@
 import { client } from '@/apollo/apolloClient';
 import { FETCH_PROJECT_BY_SLUG } from '@/apollo/gql/gqlProjects';
 import { IProject } from '@/apollo/types/types';
+import NotAvailableProject from '@/components/views/project/NotAvailableProject';
 
 import ProjectIndex from '@/components/views/project/ProjectIndex';
 
 const ProjectRoute = (props: { project?: IProject }) => {
-	return <ProjectIndex project={props.project} />;
+	return props.project ? (
```
Comment (RamRamez) [inline pages/project/[projectIdSlug]/index.tsx:9]: Have you considered `draft` projects? this way no one can view his/her draft projects anymore.

File: src/components/views/project/ProjectDonateCard.tsx [hunk 2]
```diff
@@ -276,10 +276,13 @@ const ProjectDonateCard = ({
 							When you donate to verified projects, you get GIV
 							back.
 						</Caption>
+						<GIVbackButton
+							aria-label='Learn more about this project'
+							href={links.GIVBACK_DOC}
+						>
+							Learn more
```
Comment (RamRamez) [inline src/components/views/project/ProjectDonateCard.tsx:283]: this `learn more` button doesn't exist on figma and I've removed it in PR #592

**Review by RamRamez — CHANGES_REQUESTED** [review]

**Review by mateodaza — APPROVED** [review]

**Comment by RamRamez** [comment]
@mateodaza What did you merged this PR? Please revert it.

As you see, no one can view his/her draft projects anymore.
image

**Comment by aminlatifi** [comment]
@MohammadPCh @mateodaza I had to revert it because @RamRamez comments were not addressed

**Comment by mateodaza** [comment]
@RamRamez @MohammadPCh @aminlatifi wow I totally missed that comment, my bad

**Comment by MohammadPCh** [comment]
> @MohammadPCh @mateodaza I had to revert it because @RamRamez comments were not addressed

oops!

**Pull request merged**